test/Test/Data.hs

```diff
@@ -18,32 +18,37 @@ import Hintman.Hint (getAllComments)
 
 
 data Prs = Prs
-    { pr1  :: !PrInfo
-    , pr2  :: !PrInfo
-    , pr3  :: !PrInfo
-    , pr24 :: !PrInfo
-    , pr30 :: !PrInfo
-    , pr36 :: !PrInfo
+    { pr2     :: !PrInfo
+    , pr3     :: !PrInfo
+    , pr4     :: !PrInfo
+    , pr24    :: !PrInfo
+    , pr30    :: !PrInfo
+    , pr36    :: !PrInfo
+    , forkPr1 :: !PrInfo
     }
 
-owner :: Owner
-owner = Owner "kowainik"
+kowainik :: Owner
+kowainik = Owner "kowainik"
 
 repo :: Repo
 repo = Repo "hintman-target"
 
 fetchPrs :: IO Prs
 fetchPrs = usingLoggerT richMessageAction $ do
-    pr1  <- makePr (PrNumber 2) (Sha "vrom911-patch-1")
-    pr2  <- makePr (PrNumber 3) (Sha "vrom911-patch-2")
-    pr3  <- makePr (PrNumber 4) (Sha "chshersh/parse-error")
+    pr2  <- makePr (PrNumber 2) (Sha "vrom911-patch-1")
+    pr3  <- makePr (PrNumber 3) (Sha "vrom911-patch-2")
+    pr4  <- makePr (PrNumber 4) (Sha "chshersh/parse-error")
     pr24 <- makePr (PrNumber 24) (Sha "chshersh/change-one-in-big")
     pr30 <- makePr (PrNumber 30) (Sha "chshersh/test-hintman")
     pr36 <- makePr (PrNumber 36) (Sha "chshersh-patch-1")
+    forkPr1 <- makePrFrom (Owner "popzxc") (PrNumber 1) (Sha "no-newline-at-file-end")
```

Comment:
Since you're a collaborator in the `hintman-target` repository, I propose to open a PR directly to that repo and simplify logic here :slightly_smiling_face: